core/cmd/local_client.go

```diff
@@ -73,6 +77,28 @@ func loggedStop(app chainlink.Application) {
 	logger.WarnIf(app.Stop())
 }
 
+const (
+	invalidPermissions = os.FileMode(0077)
```

Comment:
Same as below, also, is this a mask?  and not permissions?  I'm personally not sure.

Reply:
it's just a bitwise representation of the invalid permissions... ex:
```bash
# on a file you can have permissions:
rxwrxwrxw
# os.FileMode(0077) becomes
000111111
```
Then, when we use bitwise & to determine if a file have any invalid permissions
```
# overly permissive file
111000100
&
000111111
=
000000100

# good file
111000000
&
000111111
=
000000000
```